src/cwt/claim/ietf_token_status.rs

```diff
@@ -0,0 +1,151 @@
+//! Claims from the IETF Token Status List spec.
+
+use super::{Claim, Error as ClaimError, Key};
+use serde::{Deserialize, Serialize};
+use serde_cbor::Value;
+use std::collections::BTreeMap;
+
+/// Status Claim for the ReferencedToken.
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct ReferencedTokenStatus {
+    pub status_list: StatusListReference,
+}
+
+/// Claim describing the location of the Referenced Token in the Status List.
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct StatusListReference {
+    pub idx: StatusListIndex,
+    pub uri: StatusListUri,
+}
+
+/// Index to check for status information in the Status List for the Referenced Token.
+#[derive(Copy, Clone, Debug, Deserialize, Serialize, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct StatusListIndex(pub u32);
```

Comment:
It depends or whether or not I'm right about my `Claim` implementation comments below, but if so you don't really need a newtype here.